aws_lambda_builders/workflows/custom_make/utils.py

```diff
@@ -35,6 +35,15 @@ def normpath(self, path):
     def abspath(self, path):
         return os.path.abspath(path)
 
+    def isabspath(self, path):
```

Comment:
we really need to start bringing together all the utils on path, there are quite a few floating around in this repository.

Reply:
removed these utilities as it will not be used as per our discussion related to use the passed working_directory as it is, and to move the path calculation to sam cli.